src/audio/igo_nr/igo_nr.c

```diff
@@ -409,24 +383,6 @@ static int32_t igo_nr_params(struct comp_dev *dev,
 		return -EINVAL;
 	}
 
-	cd->sink_frames = dev->frames;
-	cd->source_frames = ceil_divide(dev->frames * cd->source_rate,
-					cd->sink_rate);
-
-	/* Use empirical add +10 for target frames number to avoid xruns and
-	 * distorted audio in beginning of streaming. It's slightly more than
-	 * min. needed and does not increase much peak load and buffer memory
-	 * consumption. The copy() function will find the limits and process
-	 * less frames after the buffers levels stabilize.
-	 */
-	cd->source_frames_max = cd->source_frames + 10;
-	cd->sink_frames_max = cd->sink_frames + 10;
-	cd->frames = MAX(cd->source_frames_max, cd->sink_frames_max);
```

Comment:
all the above parameters aren't set anywhere any more? If they're now unneeded - should they be removed completely?

Reply:
Yes, it's done